lib/protocol/key_params.ts

```diff
@@ -148,7 +148,7 @@ export class SNRootKeyParams {
     if (!this.content004.created) {
       return undefined;
     }
-    return new Date(this.content004.created);
+    return new Date(Number(this.content004.created));
```

Comment:
We stringify the number when we create it. This is to avoid storing large numbers on the DB side and have to worry about allocating proper column size, etc. We create it on the JS side like this: `${new Date().getTime()}`